src/components/ContactForm/index.jsx

```diff
@@ -11,18 +55,20 @@ export default function ContactForm() {
     <StyledDiv>
       <h2>Candy Fluffs Updates!</h2>
       <p>Sign up to get updates about new products and when I’ll be at anime conventions!</p>
+      <div className="form">
       <form method="post" netlify-honeypot="bot-field" data-netlify="true" name="contact" >
-        <label htmlFor="firstName">First Name</label>
-        <input type="text" name="firstName"/>
-        <label htmlFor="lastName">Last Name</label>
-        <input type="text" name="lastName"/>
-        <label htmlFor="email">Email</label>
-        <input type="text" name="email"/>
+        {/* <label htmlFor="firstName">First Name</label> */}
```

Comment:
This matches the design, but I want to confirm with BG tomorrow that she wants to not have a label for the form. Not great for accessibility.